tests/server.py

```diff
@@ -0,0 +1,80 @@
+import errno
+import subprocess
+import os
+import sys
+import socket
+from time import time as now
+import shutil
+
+server_process = None
+policy_file = os.path.join(os.path.dirname(__file__), "policy.toml")
+certificate_file = os.path.join(os.path.dirname(__file__), "host_server.pem")
+
+has_hardware_support = os.getenv("BLINDAI_TEST_NO_HW") is None
+
+
+def launch_server():
```

Comment:
Please disable the telemetry while doing the CI.

Reply:
good point